src/libs/ajax/Metrics.ts

```diff
@@ -1,15 +1,24 @@
-import axios from 'axios';
+import axios, {AxiosRequestConfig} from 'axios';
 import {getDefaultProperties} from '@databiosphere/bard-client';
 
 import {Storage} from '../storage';
 import {getBardApiUrl} from '../ajax';
 import {Token} from '../config';
+import {MetricsEventName} from 'src/libs/events';
+
+// Set default timeout for all metrics calls to 30 seconds
+const defaultSignal: AbortSignal = AbortSignal.timeout(30000);
```

Comment:
Note that this was necessary for typescriptification to make all the axios calls compile. I'm open to alternative suggestions here but this seems like a safe timeout for bard & appcues calls.